contracts/levels/AlienCodex.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.4.24;
+
+import 'zeppelin-solidity/contracts/ownership/Ownable.sol';
+
+contract AlienCodex is Ownable {
+
+  bool public contact;
+  bytes32[] public codex;
+
+  modifier contacted() {
+    assert(contact);
+    _;
+  }
+  
+  function make_contact(bytes32[] _impossible_message) public {
+    assert(_impossible_message.length > 2**200);
+    contact = true;
+  }
+
+  function push(bytes32 _content) contacted public {
```

Comment:
You could use some function names with more gravitas here too - `push`, `pop`, `edit` sort of suggest that there's an array that needs to be manipulated, whereas `record`, `retract`, `revise` (or whatever you think is cool) make me think _"Ok, I'm really interacting with an alien codex here"_.